components/library/botan/Makefile

```diff
@@ -13,58 +13,61 @@
 # Copyright 2021 Andreas Wacknitz
 #
 
-BUILD_BITS= 64_and_32
-#USE_OPENSSL11= yes
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		botan
-COMPONENT_VERSION=	1.10.17
-COMPONENT_LICENSE=	BSD
+COMPONENT_VERSION=	2.19.2
+COMPONENT_LICENSE=	BSD 2-Clause "Simplified" License
```

Comment:
I suggest to use the short SPDX identifier here.  It is more accurate and less confusing.